wandb/media.py

```diff
@@ -64,18 +65,21 @@ def transform(images, out_dir, fname):
         from PIL import Image as PILImage
         base = os.path.join(out_dir, "media", "images")
         width, height = images[0].image.size
+        if len(images) > 50:
```

Comment:
can we make this a constant since it's in 4 places and obviously I'm going to want to raise it at some point :)